doc/design/cleanup.md

```diff
@@ -0,0 +1,33 @@
+# Cleanup design
+
+Cleanup: Operations needed to do when the lvm cluster CRD is removed.
+
+## Main requirements
+
+Remove all the operator resource units and associated resources (config files, socket files). 
+This part is mostly complete and implemented in the ensureDeleted methods of the different resource units.
+
+In this document our target is to decide how to remove all the VGs created because operator working behaviour. 
+
+Main requirements are:
+- Volume Groups created by the operator must only be removed if we do not have PVCs or PVs using them.
+- If we decide that a VG need to be deleted, we must delete all the Logical Volumes in it (only one), and also the Physical Volume that supports the VG. 
+- Any solution mus be compatible with the "future". (Multiples nodes, multiple lvm cluster CRDs) 
+
+## How to implement the cleanup
+
+Two possibilities has been discussed in our [first meeting](https://drive.google.com/file/d/1ltK1XO6Y3gRPVLRLvrmsBJKjXt9s2Klw/view?usp=sharing) about this topic:
```

Comment:
- This link will not be accessible to public, we can summarise what we discussed and mention that in this doc